articles/active-directory/develop/tutorial-blazor-server.md

```diff
@@ -34,173 +34,63 @@ In this tutorial:
+dotnet tool install --global msidentity-app-sync
 ```
 
-In your browser, navigate to `https://localhost:<port number> `, and log in using an Azure AD user account to see the app running.
-
-## Retrieving data from Microsoft Graph
+This tool will automate the following tasks for you:
 
-[Microsoft Graph](/graph/overview) offers a range of APIs that provide access to your users' Microsoft 365 data. By using the Microsoft identity platform as the identity provider for your app, you have easier access to this information since Microsoft Graph directly supports the tokens issued by the Microsoft identity platform. In this section, you add code to display the signed in user's emails on the application's "fetch data" page.
+- Register your application in Azure Active Directory
+  - Create a secret for your registered application
+  - Register redirect URIs based on your launchsettings.json
+- Initialize the use of user secrets in your project
+- Store your application secret in user secrets storage
+- Update your appsettings.json with the client-id, tenant-id, etc...
 
-Before you start, log out of your app since you'll be making changes to the required permissions, and your current token won't work. If you haven't already, run your app again and select **Log out** before updating the code below.
+> .Net Tools extend the capabilities of the dotnet cli command. To learn more about .Net Tools visit: [https://learn.microsoft.com/en-us/dotnet/core/tools/global-tools](https://learn.microsoft.com/en-us/dotnet/core/tools/global-tools)
 
-Now you'll update your app's registration and code to pull a user's email and display the messages within the app. To achieve this, first extend the app registration permissions in Azure AD to enable access to the email data. Then, add code to the Blazor app to retrieve and display this data in one of the pages.
+> For more information on user secrets storage please visit: [https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-7.0&tabs=windows](https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-7.0&tabs=windows)
 
-1. In the Azure portal, select your app in **App registrations**.
-1. Under **Manage**, select **API permissions**.
-1. Select **Add a permission** > **Microsoft Graph**.
-1. Select **Delegated Permissions**, then search for and select the **Mail.Read** permission.
-1. Select **Add permissions**.
+## Use the Microsoft Identity App Sync Tool
 
-In the _appsettings.json_ file, update your code so it fetches the appropriate token with the right permissions. Add `mail.read` after the `user.read` scope under `DownstreamAPI`. This is specifying which scopes (or permissions) the app will request access to.
+Run the following command to register your app in your tenant and update the .Net configuration of your application. Provide the username/upn belonging to your Azure Account and the tenant id of the Azure Active Directory associated with your Azure Account.
```

Comment:
```suggestion
Run the following command to register your app in your tenant and update the .Net configuration of your application. Provide the username/upn belonging to your Azure Account (for instance username@domain.com) and the tenant id or domain name of the Azure Active Directory associated with your Azure Account. The account you'll use needs to be signed-in in either Visual Studio, Azure CLI, or Azure PowerShell.
```

Reply:
From what I could see in my testing.  The browser would pop if you were not already signed in.  It didn't seem like being signed into something else was a pre-req.

Reply:
I've updated the comment. Thanks @shoatman